packages/core/src/modules/vc/W3cCredentialService.ts

```diff
@@ -302,15 +303,9 @@ export class W3cCredentialService {
         }
       }
 
-      let loader
-
-      if (isNodeJS()) {
-        loader = documentLoaderNode.apply(jsonld, [])
-      } else if (isReactNative()) {
-        loader = documentLoaderXhr.apply(jsonld, [])
-      } else {
-        throw new AriesFrameworkError('Unsupported environment')
-      }
+      // fetches the documentLoader from documentLoader.ts or documentLoader.native.ts depending on the platform at bundle time
```

Comment:
The isNodeJS and isReactNative methods are defined but never used. Can we remove those methods @karimStekelenburg ?

<img width="765" alt="image" src="https://user-images.githubusercontent.com/23165168/189196270-6c8f991b-b646-4197-8ddd-1daeb96501d6.png">

Reply:
Nope, removed